components/async-operations/PROJECT

```diff
@@ -0,0 +1,7 @@
+domain: compass
```

Comment:
More about the file and its purpose can be read [here](github.com/kyma-incubator/compass/components/operations-controller). In short, this file is used by the Kubebuilder CLI to store scaffolding versions. If the file is missing, the CLI will assume that we're using a scaffolding version of `1` and since we're using version `2` this would be a false assumption. This could potentially bring some problems. So, as for now, we can't get rid of this file.